### PR TITLE
Inital commit

### PR DESCRIPTION
**Pull request by amgoodfellow**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  a soffit we don't have to reinvent the authentication wheel. Also, as added benefits, having
+  this libraries ensures a single, central place to handle changes in the future, as well as ensuring
+  consistent error reporting for every soffit's authentication step.
+
+* Including in a Project:
+  This library was made with convenience in mind. In most cases, only a single method needs to be 
+  called to verify a token, /and/ retrieve the user's personal identifier.
+  
+** Project Setup
+   After following the instructions [[https://code.oakland.edu/ea-developers/training/blob/master/module06/soffits/building-soffits.md][here]] add the following snippets to your =build.gradle=:
```
Comment (nickwoodle) [inline README.org]: We can probably remove the links to our Gitlab

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        }
+    }
+
+    repositories {
+        maven {
+          url "https://maven.oakland.edu/nexus/content/repositories/uPortal"
+          name "nexus"
+          credentials {
+            username "$mavenUser"
+            password "$mavenPassword"
```
Comment (nickwoodle) [inline build.gradle]: We can probably remove this as well

**Comment by nickwoodle** [comment]
This looks good. I think we should just remove the items I mentioned above and then I am good with merging.

File: README.org (hunk 1 above)
Comment (amgoodfellow) [reply README.org]: Nice catch

File: build.gradle (hunk 2 above)
Comment (amgoodfellow) [reply build.gradle]: True

**Pull request merged**